README.md

```diff
@@ -29,6 +29,8 @@ Open http://localhost:2030/
 
 Open http://localhost:2030/
 
+To maintain the legislation at its latest version, you can check the [auto-update tool](https://github.com/openfisca/openfisca-ops/tree/master/auto-update-pip-packages).
```

Comment:
“check the [auto-update tool]” → “set up the API server from which the legislation data will be fetched to [auto-update]”?